### PR TITLE
Added step wise guided `Add visitors to repo's README.md` project.

### PR DESCRIPTION
**Pull request by tanishq-singh-2301**
Please review it

### CODE REVIEW

**Comment by mehulmpt** [comment]
Thanks for the PR, someone would review from the team shortly

**Review by Pranavtechie — CHANGES_REQUESTED** [review]
Hey Tanishq, the structure checks out. Great Job!

Can you improve the instructions written in step0 and step1 to match the challenges given the respective steps. I feel that we can include more instructions so that the user is aware of what he needs to do.  

You can also include the Complete Overview of the project from the README.md in the step0 instructions as well. 

Also as the user is supposed to push these changes to the GitHub repository after working with  code on Codedamn Playgrounds, can you add few instructions on how to login  to a GitHub account in Codedamn Playgrounds. as currently users cannot login using passwords and have to make use of Access Tokens. 

Feel free to message back if you have questions.

**Comment by tanishq-singh-2301** [comment]
Working on it!

**Comment by tanishq-singh-2301** [comment]
Made all the changes requested.
Kindly review my code @Pranavtechie @mehulmpt

Thanks.

File: add-visitors-to-github-readme/step2/challenges.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+name: "Create a new push from workflow"
+stepId: step2
+stepBreakDown:
+  - text: "Set git global configuration (like name, email)"
+  - text: "git add README.md"
```
[the remark this answers is not in the document]
Comment (Pranavtechie) [reply add-visitors-to-github-readme/step2/challenges.yml:5]: I'm referring to the last three challenges in the step2

**Comment by tanishq-singh-2301** [comment]
Rephrased the challenges in step2,
Please review it @Pranavtechie 

Thanks.

**Comment by tanishq-singh-2301** [comment]
Any update @mehulmpt  @Pranavtechie

**Review by Pranavtechie — COMMENTED** [review]
@tanishq-singh-2301 Everything looks good. We'll merge this soon!

**Comment by tanishq-singh-2301** [comment]
Okay

**Comment by tanishq-singh-2301** [comment]
any update? @Pranavtechie  @mehulmpt

**Comment by tanishq-singh-2301** [comment]
@mehulmpt can you please review this pr?

**Comment by tanishq-singh-2301** [comment]
any update?? @Pranavtechie @mehulmpt

**Review by Pranavtechie — APPROVED** [review]

**Comment by Pranavtechie** [comment]
@tanishq-singh-2301 Please share your email address here or reach out to me at pranav@codedamn.com

**Comment by tanishq-singh-2301** [comment]
It's tanishqsingh640@gmail.com @Pranavtechie

**Pull request merged**